content/accounts.md

```diff
@@ -0,0 +1,228 @@
+
+1. **Password reset.** When the user clicks the link in their email, they are taken to a page where they can enter a new password for their account.
+1. **User enrollment.** A new user is created by an administrator, but no password is set. When the user clicks the link in their email, they are taken to a page where they can set a new password for their account. Very similar to password reset.
+1. **Email verification.** When the user clicks the link in their email, the application records that this email does indeed belong to the correct user.
+
+Here, we'll talk about how to manage the whole process manually from start to finish.
+
+#### Working email flow out of the box
+
+If you want something that just works out of the box, you can either use `accounts-ui` which does everything for you, or `useraccounts`, which we'll explain in more detail in the next section. Only follow the directions below if you definitely want to build all parts of the email flow yourself.
+
+#### Sending the email
+
+`accounts-password` comes with handy functions that you can call from the server to send an email. They are named for exactly what they do:
+
+1. [`Accounts.sendResetPasswordEmail`](http://docs.meteor.com/#/full/accounts_sendresetpasswordemail)
+2. [`Accounts.sendEnrollmentEmail`](http://docs.meteor.com/#/full/accounts_sendenrollmentemail)
+3. [`Accounts.sendVerificationEmail`](http://docs.meteor.com/#/full/accounts_sendverificationemail)
+
+The email is generated using the email templates from [Accounts.emailTemplates](http://docs.meteor.com/#/full/accounts_emailtemplates), and include links generated with `Accounts.urls`. We'll go into more detail about customizing the email content and URL later.
+
+#### Identifying when the link is clicked
+
+When the user receives the email and clicks the link inside, their web browser will take them to your app. Now, you need to be able to identify these special links and act appropriately. If you haven't customized the link URL, then you can use some built-in callbacks to identify when the app is in the middle of an email flow. When these callbacks are triggered, Meteor's accounts system enters a special state that can only be exited by calling the `done` function that is passed into the registered callback.
+
+1. [`Accounts.onResetPasswordLink`](http://docs.meteor.com/#/full/Accounts-onResetPasswordLink)
+2. [`Accounts.onEnrollmentLink`](http://docs.meteor.com/#/full/Accounts-onEnrollmentLink)
+3. [`Accounts.onEmailVerificationLink`](http://docs.meteor.com/#/full/Accounts-onEmailVerificationLink)
+
+If you have customized the URL, you will need to add a new route to your router that handles the URL you have specified.
```

Comment:
The "real" version of this will be covered in the `useraccounts` section. Here's the relevant code in todos:

https://github.com/meteor/todos/blob/c6ee2af75b3405e16f08cdd788692e3c5dd7c7c2/packages/todos-main/routes.js#L36-L39

I'm thinking I should rearrange this to put the `useraccounts` version _first_, since that's the most sane way to do things.